src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkBase;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.*;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.lib.Neo;
+
+import java.util.function.DoubleSupplier;
+
+import static frc.robot.Constants.IntakeConstants.*;
+
+public class Intake extends SubsystemBase {
+    public int targetPos = INTAKE_ANGLE.IDLE.angle;
+    private final PIDController anglePIDcontroller = new PIDController(PIDGains.kp, PIDGains.ki, PIDGains.kd);
+    private final ArmFeedforward angleFFcontroller = new ArmFeedforward(FFangleGains.ks, FFangleGains.kg, FFangleGains.kv);
+    private final Neo intakeMotor = new Neo(0);
+    private final Neo angleMotor = new Neo(0);
+    private final DutyCycleEncoder intakeEncoder = new DutyCycleEncoder(0);
+    private final DigitalInput beamBreak = new DigitalInput(0);
+    private final Trigger beamBreakTrigger = new Trigger(()-> beamBreak.get());
```

Comment:
Pleasee put some spacing between the lines.

group them by type, put the two neos together but sepereted from the rest, for example.